devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -88,27 +88,33 @@ private handleReportAttributeMessage(String description) {
     }
 }
 
-private handleBatteryEvent(rawValue) {
-    def linkText = getLinkText(device)
-
+/**
+ * Create battery event from reported battery voltage.
+ *
+ * @param volts Battery voltage in .1V increments
+ */
+private handleBatteryEvent(volts) {
     def eventMap = [
         name: 'battery',
         value: '--'
```

Comment:
not sure if this is a good idea. I have seen devices sometimes inadvertently report 0 or FF in the middle of their life cycle. Setting it to '--' causes two issues:
1. In the device tile you suddenly lose the currently battery level of the device. And probably won't get an update on its battery for probably a day
2. Battery level is tied to Low Battery Monitoring, not sure how well it is going to accept the non-int value